### ISSUE TITLE
Add information about accessibilityRole to Button documentation

### ISSUE THREAD

**Issue by tuukkao** [software-mansion/react-native-gesture-handler#1074]
currently the button documentation mentions that in order for buttons to be accessible, the button contents must be wrapped inside a `<View>` with `accessible` set to `true`. This is not enough to make the button accessible.

While this solves the issue of button text and icon appearing as two separate focusable elements, it doesn't add the necessary accessibility role to the view, making a screen reader user unable to determine that the view in question is a button. The wrapping view should also have ` accessibilityRole="button"` set.
